database/schema.ts

```diff
@@ -292,6 +292,38 @@ export const purchasesTable = mysqlTable('purchases', {
     .notNull(),
 });
 
+export const bannerJobTable = mysqlTable('banner_jobs', {
+  id: bigint('id', {
+    unsigned: true,
+    mode: 'number',
+  })
+    .autoincrement()
+    .primaryKey(),
+  user_id: mediumint('user_id', {
+    unsigned: true,
+  })
+    .references(() => userTable.id, {
+      onDelete: 'cascade',
+    })
+    .notNull(),
+  username: varchar('username', {
+    length: 32,
+  }).notNull(),
+  flair_name: varchar('flair_name', {
+    length: 24,
+  }),
+  stat_gen_time: mediumint('stat_gen_time', { unsigned: true }),
+  dragon_fetch_time: mediumint('dragon_fetch_time', { unsigned: true }),
+  dragon_gen_time: mediumint('dragon_gen_time', { unsigned: true }),
+  frame_gen_time: mediumint('frame_gen_time', { unsigned: true }),
+  gif_gen_time: mediumint('gif_gen_time', { unsigned: true }),
+  total_time: mediumint('total_time', { unsigned: true }),
+  // error: what format should error be, since it can be anything? just a string?
+  generated_on: datetime('generated_on', { mode: 'date' })
```

Comment:
```suggestion
  createdAt: datetime('created_at', { mode: 'date' })
```

just a more standardised descriptor